packages/design-system-docs/src/pages/components/FilterTag/_FilterTag.docs.scss

```diff
@@ -0,0 +1,45 @@
+/*
+Filter Tag
+
+Filter Tags are used to display dismissible filter chips
+
+Markup: filtertag.example.html
+
+Style guide: components.filtertag
+*/
+
+/*
+`<FilterTag>`
+
+@react-example FilterTag.example.jsx
+
+@react-props FilterTag.jsx
+
+Style guide: components.filtertag.react
+*/
+
+/*
+---
+
+### When to use
+
+- To filter results with one or more tags.
+
+### When to consider alternatives
+
+- When a badge is sufficient in place of a filter tag
+
+### Usage
+
+-
+
+### Accessibility
+
+-
+
+### Future research
+
+-
```

Comment:
We should add a customizations section here with SCSS variable names and what they do

Reply:
I tried to add this, but it's not showing up on the demo URL or locally. Do you know why it's not working? I copied format from a different file.